test/tools/isoltest.cpp

```diff
@@ -424,14 +425,27 @@ int main(int argc, char const *argv[])
 
 	auto& options = dynamic_cast<solidity::test::IsolTestOptions const&>(solidity::test::CommonOptions::get());
 
-	bool disableSemantics = !solidity::test::EVMHost::getVM(options.evmonePath.string());
-	if (disableSemantics)
+	bool disableSemantics = true;
+	try
 	{
-		cout << "Unable to find " << solidity::test::evmoneFilename << ". Please provide the path using --evmonepath <path>." << endl;
-		cout << "You can download it at" << endl;
-		cout << solidity::test::evmoneDownloadLink << endl;
+		disableSemantics = !solidity::test::EVMHost::checkVmPaths(options.vmPaths);
+	}
+	catch (std::exception const& _exception)
```

Comment:
What exception can be throw there?

Reply:
`std::runtime_error`, so I will only check for that one.